mesonbuild/backend/xcodebackend.py

```diff
@@ -1391,7 +1393,7 @@ def generate_xc_build_configuration(self, objects_dict: PbxDict) -> None:
             bt_dict.add_item('isa', 'XCBuildConfiguration')
             settings_dict = PbxDict()
             bt_dict.add_item('buildSettings', settings_dict)
-            settings_dict.add_item('ARCHS', '"$(NATIVE_ARCH_ACTUAL)"')
+            settings_dict.add_item('ARCHS', '"%s"' % self.arch)
```

Comment:
We have a preference for f-strings in simple cases, and `str.format()` in more complex ones. I'd prefer to see `f'"{self.arch"'}` as it's more readable and keeping with our modern style.